db/database.go

```diff
@@ -231,6 +233,7 @@ func NewDatabaseContext(dbName string, bucket base.Bucket, autoImport bool, opti
 
 	dbContext := &DatabaseContext{
 		Name:       dbName,
+		UUID:       cbgt.NewUUID(),
```

Comment:
Does this have to be a cbgt-specific UUID? (are there uuid formatting requirements where this UUID actually ends up being used?) Or can it be one from the other UUID package we use?

Reply:
This was mainly out of convenience.  It is required that the UUID we use match the one we pass to CBGT, unless we want to build a mapping from our UUID to the cbgt UUID into the import listener.  (the latter isn't out of the question when we can't reuse, but I'm avoiding unless actually necessary)

As far as I can tell CBGT doesn't require a UUID format.  I'm not against changing to uuid.NewRandom(), but not sure it's a priority given the relatively low number of UUIDs and associated collision risk in this usage.

Reply:
Yep fair enough, especially given it's only a basic util function in cbgt, and not pulling any other cbgt stuff into here.